README.md

```diff
@@ -2,10 +2,10 @@
 
 The purpose of this Ansible collection is to automate the deployment of the Red Hat Trusted Artifact Signer (RHTAS) service on Red Hat Enterprise Linux (RHEL).
 
-> [!IMPORTANT]
+**IMPORTANT:**
```

Comment:
I think we should remove this notice altogether now, as we won't be shipping this as tech preview in 1.2.0.